distribution/docs/src/main/resources/content/_securityFramework/federated-identity.adoc

```diff
@@ -0,0 +1,65 @@
+requests, allowing federated sources to run queries as that user. This is critical because {branding}
+must make authorization decisions for all queries, not just those initiated by the local node.
+
+To represent a user's identity when federating, {branding} uses a SAML assertion. This is the format
+used internally to represent all subjects (even unauthenticated subjects). When federating over REST,
+the assertion is simply attached as an HTTP header to the query request, and extracted by the
+federated source when the request is received. If the assertion is signed by a trusted cert, the
+federated source will then run the query as the user.
+
+The figure below shows a federated query between two instances of {branding} that support
+federated identity.
+
+[ditaa, federated_identity, png, ${image-width}]
+....
+                  /---------------------------\           /--------------------------\
+                  | cDEF       DDF            |           |     Federated Source     |
+/--------\        | +-----------------------+ |           | cDEF                     |
+|  User  +------->| |    Send request to    | |           | +----------------------+ |
+|  cDEF  | Search | |   Catalog Framework   | |  /------->| |   Recreate subject   | |
+\--------/        | +-----------------------+ |  | Query  | +----------------------+ |
+     ^            |             |             |  |        |             |            |
+     |            |             v             |  |        |             v            |
+     |            | +-----------------------+ |  |        | +----------------------+ |
+     |            | | Create HTTP request & | +--/        | | Run query as subject | |
+     |            | | attach SAML assertion | |           | +----------------------+ |
+     |            | +-----------------------+ |           |             |            |
+     |            |                           |           |             v            |
+     |            | +-----------------------+ |           | +----------------------+ |
+     \------------+ |     Filter results    | |<----------+ |    Filter results    | |
+                  | +-----------------------+ |  Filtered | +----------------------+ |
+                  \---------------------------/   results \--------------------------/
+....
+
+. The user submits a search to {branding}.
+. {branding} generates a catalog request and attaches the user's <<_subject,Subject>>. This
+Subject stores a SAML assertion, which is generated and signed by the
+<<_security_token_service, STS>>. The catalog request gets sent to the Catalog Framework.
+. The Catalog Framework extracts the SAML assertion stored by the Subject and sends an HTTP request
+to each federated source with the assertion included. When federating over REST, it places the
+assertion in the Authorization HTTP header.
+. A federated source receives this request and extracts the SAML assertion. It attempts to validate
```

Comment:
The pronoun "It" in "It attempts to validate" might be ambiguous to the reader. Replace "It" with the referent.

`The Federated Source validates the authenticity of the SAML Assertion. If the Assertion if valid, the Federated Source generated a Subject from the Assertion to represent the user who initiated the request.`